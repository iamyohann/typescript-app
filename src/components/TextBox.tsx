import * as React from 'react';

const TextBox: React.SFC<React.HTMLAttributes<HTMLInputElement>> = (props: React.HTMLAttributes<HTMLInputElement>) => {
    return <input {...props} />
}

export default TextBox;
