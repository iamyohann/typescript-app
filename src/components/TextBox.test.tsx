import * as React from 'react';
import * as ReactDOM from 'react-dom';
import * as renderer from 'react-test-renderer';

import TextBox from './TextBox';

it('renders according to the snapshot', () => {
  const tree = renderer
    .create(<TextBox />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});