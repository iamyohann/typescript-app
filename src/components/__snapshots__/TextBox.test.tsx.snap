// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders according to the snapshot 1`] = `<input />`;
